src/adapter/breakpoints.ts

```diff
@@ -255,6 +288,20 @@ export class BreakpointManager {
       }
 
       await Promise.all(todo);
+
+      return {
```

Comment:
As I pointed before, this could be probably simplified, without the need to track attachedLoactions:

```
<here>
const remainPaused = (await Promise.all(todo)).some(x => !!x);
</here>

async updateForSourceMap(thread: Thread, script: Script): Promise<boolean> {
   ...
  return uiLocations.any(loc => !loc.lineNumber && !loc.columnNumber);
}
```

Reply:
Ah, I see what you mean. That seems to work.